src/_guides/whats-new.md

```diff
@@ -17,6 +17,107 @@ and follow the [Dart blog][].
+For details about the 2.19 + 3.0 alpha releases,
+see [Dart FlutterForward + 3.0 alpha blog post][],
+and the [SDK changelog][].
+
+[Dart FlutterForward + 3.0 alpha blog post]: https://medium.com/dartlang/dart-3-alpha-f1458fb9d232
+[SDK changelog]: https://github.com/dart-lang/sdk/blob/master/CHANGELOG.md#2190
+
+### Docs updated or added to dart.dev
+{: .no_toc}
+
+In addition to bug fixes and incremental improvements,
+we made the following changes to this site:
+
+* Introduced the [Fetch data from the internet][] tutorial about using `package:http`.
+* Added a page on [Automated publishing of packages to pub.dev][].
+* Included links to three new site translations in our [community resources section][]:
+  * [Korean version of this site (한국어)](https://dart-ko.dev/)
+  * [Simplified Chinese version of this site (简体中文版)](https://dart.cn)
+  * [Traditional Chinese version of this site (正體中文版)](https://dart.tw.gh.miniasp.com/)
+* Updated null safety content in preparation of Dart 3:
+  * Changed the version constraints in the [migration guide][] for Dart 3 compatibility.
+  * Added Dart 3 full sound null safety overview to the [Sound null safety][] page.
+  * Emphasized Dart 3's incompatibility with [unsound null safety][] in a note.
+* Introduced the new [Learning Dart as a Swift developer][] guide.
+* Replaced an Effective Dart section with more general guidance on [booleans and equality operators][]. 
+* Documented [content-hashing][] across the pub docs.
+* Began effort to overhaul [Zones][] page by changing examples to use `runZonedGuarded` instead of `onError`.
+* Updated content on libraries to cover new no-name declarations development:
+  * Effective Dart: [Documentation][], [Style][], and [Usage][]
+  * New library directive section in [The language tour][]
+* Improved clarity surrounding Dart's single-threaded or multi-threaded status:
+  * Removed the outdated `dart:io` page.
+  * Added two sections to the FAQ:
+    * [Is Dart single-threaded?][]
+    * [Is Dart single-threaded on the web?][]
+  * Expanded on [Dart's web concurrency capabilities][].
+* Rearranged and clarified [discussion][] of default values for optional and positional parameters.
+* Updated [Concurrency in Dart][] to default to new `Isolate.run()` function.
+* Documented specifying a file path when activating a package on the [`pub global` page][].
+* Rewrote [Learning Dart as a JavaScript developer][].
+* Added a brief overview of Dart DevTools to [`dart run` page][].
+* Provided more clarity around [operator precedence and associativity][] in the Language tour.
+* Expanded Library tour section on [Building URIs][] with URI http and factory constructor info.
+* Accounted for [pub's transition to pub.dev][] from pub.dartlang.org.
+* Added documentation on [package screenshots][].
+* Improved the [explicit downcast section][] of The Dart type system page.
+* Increased [analyzer][] and [lint][] coverage:
+  * Included SDK information for linter rules.
```

Comment:
Could we be more specific this about which SDK versions support the rules? Maybe:

```suggestion
  * Included SDK version support for linter rules.
```